File: Examen/C/Factorial.c
#include <stdio.h>

int main() {
    int n, i;
    long fact = 1;

    printf("Ingrese un numero entero: ");
    scanf("%d", &n);

    for(i = 1; i <= n; i++)
        fact *= i;

    printf("El factorial de %d es: %ld\n", n, fact);

    return 0;
}

File: Examen/Fortran/Factorial.f
      PROGRAM Factorial
      INTEGER N, I
      INTEGER FACT

      PRINT *, 'Ingrese un numero entero:'
      READ *, N

      FACT = 1
      DO 10 I = 1, N
         FACT = FACT * I
10    CONTINUE

      PRINT *, 'El factorial de ', N, ' es: ', FACT

      PRINT *, 'Presione ENTER para salir...'
      READ(*,*)
      END

